testsuite/classlibrary/TestReadableNodeIDAllocator.sc

```diff
@@ -0,0 +1,85 @@
+TestReadableNodeIDAllocator : UnitTest {
+
+	// test proper creation from Server
+	test_readableNodeRange {
+		var s = Server(\testRNIA);
+		var prevClass = Server.nodeAllocClass;
+		Server.nodeAllocClass = ReadableNodeIDAllocator;
+		s.options.maxLogins = 1; s.newAllocators;
```

Comment:
reminder to put these on separate lines or move into a helper function within this class

Reply:
yes - added Server:remove method, as it is generally useful be able to completely remove a server in a sinlgle clear method.